meson.build

```diff
@@ -885,60 +897,66 @@ if rpc_headers_ok
     endforeach
 endif
 
-if with_quota.disabled()
+if not enable_quota
     have_quota = false
     cdata.set('NO_QUOTA_SUPPORT', 1)
-elif tirpc.found() and cc.has_header('rpcsvc/rquota.h')
-    have_quota = true
-    cdata.set('NEED_RQUOTA', 1)
-    quota_deps += tirpc
-    quota_provider += 'libtirpc'
-    if cc.compiles(
-        '''
-                #include <rpcsvc/rquota.h>
-                int main(void) {
-                    enum qr_status foo;
-                    foo = Q_OK;
-                    return 0;
-                }
-            ''',
-        dependencies: tirpc,
-    )
-        cdata.set('HAVE_RQUOTA_H_QR_STATUS', 1)
-    endif
-elif rpcsvc.found() and rpc_headers_ok
-    have_quota = true
-    quota_deps += rpcsvc
-    quota_provider += 'SunRPC'
-    if quota.found() and cc.has_function('getfsquota', dependencies: [quota, prop, rpcsvc])
-        quota_deps += [quota, prop]
-        cdata.set('HAVE_LIBQUOTA', 1)
-    endif
 else
-    have_quota = false
-    cdata.set('NO_QUOTA_SUPPORT', 1)
+    have_quota = tirpc.found() and cc.has_header('rpcsvc/rquota.h')
+    if have_quota
+        cdata.set('NEED_RQUOTA', 1)
+        quota_deps += tirpc
+        quota_provider += 'libtirpc'
+        if cc.compiles(
+            '''
+                    #include <rpcsvc/rquota.h>
+                    int main(void) {
+                        enum qr_status foo;
+                        foo = Q_OK;
+                        return 0;
+                    }
+                ''',
```

Comment:
Nitpick: Close of multiline string has uneven indentation compared to the beginning.

Reply:
All these formatting issues will be fixed once meson 1.5.0 is released. At this stage I will meson format all the meson.build files so their formatting is entirely consistent with meson guidelines.